src/components/items/Container.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import Fetch from 'utilities/fetch';
+import ItemCard from '../item_card/Presenter';
+import ItemsPagePresenter from './Presenter';
+
+const ItemsPage = () => {
+    const [items, setItems] = useState([]);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        const fetchItems = async () => {
+            let result = await Fetch("/api/items");
+            let { data, error } = result;
+            data ? setItems(data) : setError(error);
+        }
+        fetchItems();
+    }, []);
+
+    const createItemComponents = (data) => {
+        return data.map((object, index) => {
```

Comment:
It's an [anti-pattern](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) to use the index when creating component lists. I'd suggest using some other unique identifier like the item's id.